src/fast_layers.jl

```diff
@@ -35,6 +35,21 @@ struct FastDense{F,F2} <: FastLayer
 end
 # (f::FastDense)(x,p) = f.σ.(reshape(uview(p,1:(f.out*f.in)),f.out,f.in)*x .+ uview(p,(f.out*f.in+1):lastindex(p)))
 (f::FastDense)(x,p) = f.σ.(reshape(p[1:(f.out*f.in)],f.out,f.in)*x .+ p[(f.out*f.in+1):end])
+ZygoteRules.@adjoint function (f::FastDense)(x,p)
+  W = p[reshape(1:(f.out*f.in),f.out,f.in)] # @view after BLAS fixes
+  b = p[(f.out*f.in+1):end]

```

Comment:
I'd imagine this one could be a view for a small gain.  Not as big an impact as the bigger matrix, though, of course.

Reply:
Both ended up being slower with views 🤷‍♂